pages/pipelines/audit_log.md

```diff
@@ -100,3 +105,146 @@ USER_UPDATED
+
+```
+{
+  "data" => {
+    "auditEvent" => {
+      "__typename" => "AuditEvent",
+      "id" => "QXVkaXRFdmVudC0tLTAxOGUzZjE5LTlkODgtNDBmZS1iOGIzLTkxMTk5OWNlMmRmMg==",
+      "uuid" => "018e3f19-9d88-40fe-b8b3-911999ce2df2",
+      "type" => "SECRET_READ",
+      "subject" => {
+        "id" => "QXVkaXRTdWJqZWN0LS0tMDE4ZTNmMTktOWQ4OC00MGZlLWI4YjMtOTExOTk5Y2UyZGYy",
+        "type" => "SECRET",
+        "uuid" => "644771e7-10cf-4784-af97-9fdf70402a1c",
+        "node" => {
+          "__typename" => "Secret",
+          "uuid" => "644771e7-10cf-4784-af97-9fdf70402a1c",
+          "organization" => {
+            "name" => "Sunny Spot"
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+* `SECRET_QUERIED` triggers an audit event when a user belonging to an organization or system identity (such as an agent) when a query is performed to find a secret (or secrets). This event will be triggered even if a search for a secret yields no results or if the secret does not exist. Below are the fields captured in the audit log for this event.
+
+```
+{
+  "data" => {
+    "auditEvent" => {
+      "__typename" => "AuditEvent",
+      "id" => "QXVkaXRFdmVudC0tLTAxOGUzZjRlLTdiNGUtNDQ1ZS04MDI3LWQyZGU4ZjY3MDI0Yg==",
+      "uuid" => "018e3f4e-7b4e-445e-8027-d2de8f67024b",
+      "type" => "SECRET_QUERIED",
+      "subject" => {
+        "id" => "QXVkaXRTdWJqZWN0LS0tMDE4ZTNmNGUtN2I0ZS00NDVlLTgwMjctZDJkZThmNjcwMjRi",
+        "type" => "SECRET",
+        "uuid" => "d906f471-92a9-4725-aad5-d7388280e654",
+        "node" => {
+          "__typename" => "Secret",
+          "uuid" => "d906f471-92a9-4725-aad5-d7388280e654",
+          "organization" => {
+            "name" => "Sunny Spot"
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+* `SECRET_UPDATED` triggers an audit event whenever a user within an organization updates the value or properties of a secret. Given that secrets can exist in multiple versions, the audit logs maintain records of these version identifiers and their corresponding updates. Below are the fields captured in the audit log for this event.
+
+```
+{
+  "data" => {
+    "auditEvent" => {
+      "__typename" => "AuditEvent",
+      "id" => "QXVkaXRFdmVudC0tLTAxOGUzZjUxLWViNTMtNGVlZC1hZmRjLWE2ZTdhZjcyMDFkOQ==",
+      "uuid" => "018e3f51-eb53-4eed-afdc-a6e7af7201d9",
+      "type" => "SECRET_UPDATED",
+      "subject" => {
+        "id" => "QXVkaXRTdWJqZWN0LS0tMDE4ZTNmNTEtZWI1My00ZWVkLWFmZGMtYTZlN2FmNzIwMWQ5",
+        "type" => "SECRET",
+        "uuid" => "87a44525-2a66-441a-89f6-8b559364aed9",
+        "node" => {
+          "__typename" => "Secret",
+          "uuid" => "87a44525-2a66-441a-89f6-8b559364aed9",
+          "organization" => {
+            "name" => "Sunny Spot"
+          }
+        }
+      }
+    }
+  }
+}
+```
```

Comment:
I'm not sure that this info belongs here, given that we don't list it here for any of the other audit event types in this section. I would have thought that the GraphQL part of our docs would cover this sort of thing

Reply:
I agree with you. I wasn't 100% sure where it should go.
See the discussion here on the [Linear Ticket](https://linear.app/buildkite/issue/COMP-264/pre-release-secrets-audit-logs#comment-a792587c)

I might have gone in the wrong direction

Reply:
> If you wanted to add a comment on that section

Maybe that meant the doc ticket, and not the Audit Log section o.0

Reply:
have removed and popped those docs as a comment in mentioned linear ticket